site/content/blog/2022-08-30-spider-move/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Spider News"
+summary: >
+  News about changes to the Traditional Spider for the up-coming release.
+images:
+- https://www.zaproxy.org/blog/2022-08-30-spider-move/images/zapbot.spider.jpg
+addSocialPreview: true
+type: post
+date: "2022-08-30"
+authors:
+    - thorin
+---
+
+# Traditional Spider News
+
+For the upcoming stable release of ZAP a number of changes have been made to the Traditional Spider which are currently available for testing in the weekly releases.
+
+## Moved to an Add-on
+
+To facilitate more frequent functional enhancements and bug fixes the core Spider has been moved to an add-on which means such changes are no longer bound to core/stable releases. Other add-ons which use Traditional Spider functionality have also been re-worked to support the Spider add-on, including: QuickStart, FormHandler, GraphQL, OpenAPI, SOAP, and the Automation Framework. Please help us test this out by using a [weekly release](/download/#weekly).
+
+## Functional Improvements for 2.12
+
+A large number of functional improvements have been made since 2.11. Most of which have been a result of efforts related to [Issue 7152](https://github.com/zaproxy/zaproxy/issues/7152) which is backed by bounties via the Stackhawk [ZAP Fund](https://www.stackhawk.com/zap-fund/).
+Please feel free to jump in and claim some bounties yourself! The following table lists the changes in the Spider's behaviour after said updates.
```

Comment:
Also link to https://www.zaproxy.org/docs/scans/crawlmaze/ and point out that the score has gone from 50% to 80% ?

Reply:
Ha, just seen you do that below :P